modules/infra/datapath/rx.c

```diff
@@ -52,10 +53,15 @@ rx_process(struct rte_graph *graph, struct rte_node *node, void **objs, uint16_t
 			rte_node_enqueue(graph, node, NO_IFACE, &node->objs[count], rx);
 			continue;
 		}
-		for (int r = count; r < count + rx; r++) {
+		for (r = count; r < count + rx; r++) {
 			eth_input_mbuf_data(node->objs[r])->iface = iface;
-			trace_packet("rx", iface->name, node->objs[r]);
 		}
+		if (unlikely(packet_trace_enabled)) {
+			for (r = count; r < count + rx; r++) {
+				trace_packet("rx", iface->name, node->objs[r]);
```

Comment:
Nit: could we have trace_packets() that takes an array of packets?

Reply:
I guess we could, but it would be used only in rx.c. In eth_output, packets are printed one by one.

Reply:
Ok, let's wait for another user to appear.